config_dev.toml.example

```diff
@@ -102,27 +102,28 @@ INTERNAL_IPS=127.0.0.1
 # reports. It is used to separate deployments within Sentry UI
 SENTRY_ENVIRONMENT=local-development-unconfigured
 
-# Kerrokantasi uses JWT tokens for authentication. This settings Specifies
-# the value that must be present in the "aud"-key of the token presented
-# by a client when making an authenticated request. Kerrokantasi uses this
-# key for verifying that the token was meant for accessing this particular
-# Kerrokantasi instance (the tokens are signed, see below).
-# Does not correspond to standard Django setting
-TOKEN_AUTH_ACCEPTED_AUDIENCE=string-identifying-this-kerrokantasi-instance
-
-# This key is used by Kerrokantasi to verify the JWT token is from trusted
-# Identity Provider (OpenID terminology). The provider must have signed
-# the JWT TOKEN using this shared secret
-# Does not correspond to standard Django setting
-TOKEN_AUTH_SHARED_SECRET=abcdefghacbdefgabcdefghacbdefgabcdefghacbdefgabcdefghacbdefgabcdefghacbdefg
```

Comment:
@Jogha This is still needed. It uses it in JWT_AUTH. See: 
- https://github.com/City-of-Helsinki/kerrokantasi/blob/a123d2265a839ab05ae2fbf9273c343198400f57/kerrokantasi/settings/base.py#L107
- https://github.com/City-of-Helsinki/kerrokantasi/blob/5d74acafe7ac710dbfad5d12d6586196c2c04de0/kerrokantasi/settings/__init__.py#L8